2023-04-24 16:27:25,907 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Configure stats pid to 2931530
2023-04-24 16:27:25,935 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Loading settings from /home1/07169/wgl/.config/wandb/settings
2023-04-24 16:27:25,936 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Loading settings from /scratch/07169/wgl/adjoint/python/wandb/settings
2023-04-24 16:27:25,937 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-04-24 16:27:25,938 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-04-24 16:27:25,938 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-04-24 16:27:25,939 INFO    MainThread:2931530 [wandb_init.py:_log_setup():507] Logging user logs to /scratch/07169/wgl/adjoint/python/wandb/run-20230424_162725-axqc6qb9/logs/debug.log
2023-04-24 16:27:25,940 INFO    MainThread:2931530 [wandb_init.py:_log_setup():508] Logging internal logs to /scratch/07169/wgl/adjoint/python/wandb/run-20230424_162725-axqc6qb9/logs/debug-internal.log
2023-04-24 16:27:25,941 INFO    MainThread:2931530 [wandb_init.py:init():547] calling init triggers
2023-04-24 16:27:25,941 INFO    MainThread:2931530 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-04-24 16:27:25,942 INFO    MainThread:2931530 [wandb_init.py:init():595] starting backend
2023-04-24 16:27:25,942 INFO    MainThread:2931530 [wandb_init.py:init():599] setting up manager
2023-04-24 16:27:25,955 INFO    MainThread:2931530 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-04-24 16:27:25,969 INFO    MainThread:2931530 [wandb_init.py:init():605] backend started and connected
2023-04-24 16:27:25,981 INFO    MainThread:2931530 [wandb_init.py:init():695] updated telemetry
2023-04-24 16:27:26,038 INFO    MainThread:2931530 [wandb_init.py:init():732] communicating run to backend with 60.0 second timeout
2023-04-24 16:27:26,304 INFO    MainThread:2931530 [wandb_run.py:_on_init():2176] communicating current version
2023-04-24 16:27:26,481 INFO    MainThread:2931530 [wandb_run.py:_on_init():2185] got version response 
2023-04-24 16:27:26,482 INFO    MainThread:2931530 [wandb_init.py:init():782] starting run threads in backend
2023-04-24 16:27:26,812 INFO    MainThread:2931530 [wandb_run.py:_console_start():2157] atexit reg
2023-04-24 16:27:26,813 INFO    MainThread:2931530 [wandb_run.py:_redirect():2012] redirect: SettingsConsole.WRAP_RAW
2023-04-24 16:27:26,814 INFO    MainThread:2931530 [wandb_run.py:_redirect():2077] Wrapping output streams.
2023-04-24 16:27:26,815 INFO    MainThread:2931530 [wandb_run.py:_redirect():2102] Redirects installed.
2023-04-24 16:27:26,816 INFO    MainThread:2931530 [wandb_init.py:init():824] run started, returning control to user process
2023-04-24 16:27:26,818 INFO    MainThread:2931530 [wandb_config.py:__setitem__():152] config set problem = ResNet - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x1528597a9af0>>
2023-04-24 16:27:26,818 INFO    MainThread:2931530 [wandb_run.py:_config_callback():1285] config_cb problem ResNet None
2023-04-24 16:27:26,819 INFO    MainThread:2931530 [wandb_config.py:__setitem__():152] config set method = final_value_loss - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x1528597a9af0>>
2023-04-24 16:27:26,819 INFO    MainThread:2931530 [wandb_run.py:_config_callback():1285] config_cb method final_value_loss None
2023-04-24 16:28:10,671 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Configure stats pid to 2931530
2023-04-24 16:28:10,793 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Loading settings from /home1/07169/wgl/.config/wandb/settings
2023-04-24 16:28:10,795 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Loading settings from /scratch/07169/wgl/adjoint/python/wandb/settings
2023-04-24 16:28:10,797 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-04-24 16:28:10,798 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-04-24 16:28:10,799 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-04-24 16:28:10,800 INFO    MainThread:2931530 [wandb_init.py:_log_setup():507] Logging user logs to /scratch/07169/wgl/adjoint/python/wandb/run-20230424_162810-ijl1v5at/logs/debug.log
2023-04-24 16:28:10,802 INFO    MainThread:2931530 [wandb_init.py:_log_setup():508] Logging internal logs to /scratch/07169/wgl/adjoint/python/wandb/run-20230424_162810-ijl1v5at/logs/debug-internal.log
2023-04-24 16:28:10,803 INFO    MainThread:2931530 [wandb_init.py:init():547] calling init triggers
2023-04-24 16:28:10,804 INFO    MainThread:2931530 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-04-24 16:28:10,805 INFO    MainThread:2931530 [wandb_init.py:init():590] wandb.init() called when a run is still active
2023-04-24 16:28:10,807 INFO    MainThread:2931530 [wandb_config.py:__setitem__():152] config set problem = ResNet - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x1528597a9af0>>
2023-04-24 16:28:10,808 INFO    MainThread:2931530 [wandb_run.py:_config_callback():1285] config_cb problem ResNet None
2023-04-24 16:28:10,810 INFO    MainThread:2931530 [wandb_config.py:__setitem__():152] config set method = final_value_loss - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x1528597a9af0>>
2023-04-24 16:28:10,811 INFO    MainThread:2931530 [wandb_run.py:_config_callback():1285] config_cb method final_value_loss None
2023-04-24 16:30:26,086 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Configure stats pid to 2931530
2023-04-24 16:30:26,148 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Loading settings from /home1/07169/wgl/.config/wandb/settings
2023-04-24 16:30:26,151 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Loading settings from /scratch/07169/wgl/adjoint/python/wandb/settings
2023-04-24 16:30:26,153 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-04-24 16:30:26,155 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-04-24 16:30:26,157 INFO    MainThread:2931530 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-04-24 16:30:26,159 INFO    MainThread:2931530 [wandb_init.py:_log_setup():507] Logging user logs to /scratch/07169/wgl/adjoint/python/wandb/run-20230424_163025-1kvotv24/logs/debug.log
2023-04-24 16:30:26,161 INFO    MainThread:2931530 [wandb_init.py:_log_setup():508] Logging internal logs to /scratch/07169/wgl/adjoint/python/wandb/run-20230424_163025-1kvotv24/logs/debug-internal.log
2023-04-24 16:30:26,163 INFO    MainThread:2931530 [wandb_init.py:init():547] calling init triggers
2023-04-24 16:30:26,165 INFO    MainThread:2931530 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-04-24 16:30:26,166 INFO    MainThread:2931530 [wandb_init.py:init():590] wandb.init() called when a run is still active
2023-04-24 16:30:26,168 INFO    MainThread:2931530 [wandb_config.py:__setitem__():152] config set problem = ResNet - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x1528597a9af0>>
2023-04-24 16:30:26,169 INFO    MainThread:2931530 [wandb_run.py:_config_callback():1285] config_cb problem ResNet None
2023-04-24 16:30:26,171 INFO    MainThread:2931530 [wandb_config.py:__setitem__():152] config set method = final_value_loss - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x1528597a9af0>>
2023-04-24 16:30:26,173 INFO    MainThread:2931530 [wandb_run.py:_config_callback():1285] config_cb method final_value_loss None
2023-04-24 16:31:59,113 WARNING MsgRouterThr:2931530 [router.py:message_loop():77] message_loop has been closed
